akka-projection-grpc/src/main/scala/akka/projection/grpc/consumer/GrpcReadJournalProvider.scala

```diff
@@ -15,10 +15,10 @@ import com.typesafe.config.Config
  */
 final class GrpcReadJournalProvider(system: ExtendedActorSystem, config: Config, cfgPath: String)
     extends ReadJournalProvider {
-  override val scaladslReadJournal: scaladsl.GrpcReadJournal =
+  override def scaladslReadJournal(): akka.persistence.query.scaladsl.ReadJournal =
     new scaladsl.GrpcReadJournal(system, config, cfgPath)
```

Comment:
if it's not possible to have the `val` here we should still have a `private val` that the `def` returns, because we don't want to create a new instance each time

Reply:
Will review once the actual scala3 pr will come.